workers/unity/Packages/com.improbable.gdk.deploymentlauncher/Commands/Deployment.cs

```diff
@@ -15,7 +15,7 @@ public static class Deployment
             Tools.Common.GetPackagePath("com.improbable.gdk.deploymentlauncher"),
             ".DeploymentLauncher/DeploymentLauncher.csproj"));
 
-        public static WrappedTask<Result<RedirectedProcessResult, Ipc.Error>, BaseDeploymentConfig> LaunchAsync(string projectName, string assemblyName, BaseDeploymentConfig config)
+        public static WrappedTask<Result<RedirectedProcessResult, Ipc.Error>, (string, string, BaseDeploymentConfig)> LaunchAsync(string projectName, string assemblyName, BaseDeploymentConfig config)
```

Comment:
A concrete return type may be more readable...

Reply:
Agreed - this was a workaround to get the auth stuff working. I'm gonna take another pass over it today as it made things.. messy..